src/layout/organize/JourneyInstanceLayout.tsx

```diff
@@ -61,8 +56,18 @@ const JourneyInstanceLayout: React.FunctionComponent = ({ children }) => {
             }
           );
         },
-      })),
-  });
+      })) ?? [];
+
+  if (subMenuItems.length) {
+    ellipsisMenu.push({
+      id: 'convert-journey',
+      label: intl.formatMessage({
+        id: 'pages.organizeJourneyInstance.ellipsisMenu.convert',
+      }),
+      startIcon: <Forward color="secondary" />,
+      subMenuItems,
+    });
+  }
```

Comment:
This is necessary to fix the bug that was causing the tests to fail. It basically says:

* If `journeys` is undefined (on line 40), `subMenuItems` should be an empty array
* Only add the "Convert to…" option if `subMenuItems` is non-empty

This is nice because it checks two things: That the journeys have loaded, and that there exists more journeys than the one for this instance (because otherwise it would have been filtered out and `subMenuItems` would still be left empty).